packages/assertions/assertions/td-vocab-contentEncoding--StringSchema.json

```diff
@@ -0,0 +1,79 @@
+{
+    "title": "td-vocab-contentEncoding--StringSchema",
+    "description": "contentEncoding: Specifies the encoding used to store the contents, as specified in RFC 2054. The values that are accepted: \"7bit\", \"8bit\", \"binary\", \"quoted-printable\" and \"base64\". MAY be included. Type: string.",
+    "$schema ": "http://json-schema.org/draft/2019-09/schema#",
+    "is-complex": true,
+    "type": "object",
+    "properties": {
+        "properties": {
+            "type": "object",
+            "additionalProperties": {
+                "$ref": "#/$defs/property_element"
+            }
+        },
+        "actions": {
+            "type": "object",
+            "additionalProperties": {
+                "$ref": "#/$defs/action_element"
+            }
+        },
+        "events": {
+            "type": "object",
+            "additionalProperties": {
+                "$ref": "#/$defs/event_element"
+            }
+        }
+    },
+    "additionalProperties": true,
+    "$defs": {
+        "property_element": {
+            "$ref": "#/$defs/dataSchema"
+        },
+        "action_element": {
+            "type": "object",
+            "properties": {
+                "input": {
+                    "$ref": "#/$defs/dataSchema"
+                },
+                "output": {
+                    "$ref": "#/$defs/dataSchema"
+                }
+            }
+        },
+        "event_element": {
+            "type": "object",
+            "properties": {
+                "subscription": {
+                    "$ref": "#/$defs/dataSchema"
+                },
+                "data": {
+                    "$ref": "#/$defs/dataSchema"
+                },
+                "cancellation": {
+                    "$ref": "#/$defs/dataSchema"
+                }
+            }
+        },
+        "dataSchema": {
+            "type": "object",
+            "if": {
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "enum": [
+                            "string"
+                        ]
+                    },
+                    "contentEncoding": {
+                        "type": "string",
+                        "enum": ["7bit", "8bit", "binary", "quoted-printable", "base64"]
```

Comment:
these are only examples, not mandatory values. removing enum is needed

Reply:
I copied this from the schema

Reply:
Yes this should be removed